### PR TITLE
Fixed product comparison table

### PR DESCRIPTION
**Pull request by kavita-p**

### CODE REVIEW

File: client/src/components/Related/ComparisonModal.jsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const ComparisonModal = (props) => {
       onClick={closeModal}>
       {/* <button onClick={closeModal}>X</button> */}
```
Comment (ryanaolea) [inline client/src/components/Related/ComparisonModal.jsx]: saving or testing with this?

Comment (kavita-p) [reply client/src/components/Related/ComparisonModal.jsx]: oh i'll get rid of that real quick, i couldn't get the close button to work but it doesn't need it

Comment (kavita-p) [reply client/src/components/Related/ComparisonModal.jsx]: fixed!

File: client/src/components/Related/ProductCard.jsx [hunk 2]
```diff
@@ -75,7 +74,7 @@ const ProductCard = (props) => {
         <StarsTile stars ={starAverage} />
       </div>
       <button onClick={openComparisonModal}>show comparison window</button>
```
Comment (ryanaolea) [inline client/src/components/Related/ProductCard.jsx]: maybe take out the window word, so button reads: show comparison. Sounds nicer.

**Review by ryanaolea — CHANGES_REQUESTED** [review]
NIce work getting this done! Just a few tweeks/ suggestions.

File: client/src/components/Related/ComparisonModal.jsx [hunk 3]
```diff
@@ -13,9 +13,8 @@ const ComparisonModal = (props) => {
   return ReactDom.createPortal(
     <div className='pic-modal-container' ref ={modalRef}
       onClick={closeModal}>
-      {/* <button onClick={closeModal}>X</button> */}
```
Comment (ryanaolea) [inline client/src/components/Related/ComparisonModal.jsx:16]: saving for testing?

File: client/src/components/Related/ProductCard.jsx (hunk 2 above)
Comment (kavita-p) [reply client/src/components/Related/ProductCard.jsx]: fixed

**Review by ryanaolea — APPROVED** [review]

**Pull request merged**